### PR TITLE
Hello GLFW Window

### PR DESCRIPTION
**Pull request by AlexB8675**

### CODE REVIEW

File: src/Cpp/1-getting-started/12-HelloGLFWWindow/Main.cpp [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    glfwSetWindowPos(
+        windowHandle,
+        videoMode->width / 2 - currentWidth / 2,
+        videoMode->height / 2 - currentHeight / 2);
+}
+
+int main()
+{
+    const int32_t windowWidth = 1280;
+    const int32_t windowHeight = 720;
```
Comment (Themperror) [inline src/Cpp/1-getting-started/12-HelloGLFWWindow/Main.cpp:20]: Please do not hardcore any resolution, retrieve the current desktop resolution and use 90% of it instead

**Review by Themperror — CHANGES_REQUESTED** [review]

File: src/Cpp/1-getting-started/12-HelloGLFWWindow/Main.cpp (hunk 1 above)
Comment (AlexB8675) [reply src/Cpp/1-getting-started/12-HelloGLFWWindow/Main.cpp]: Understood, I have fixed the issue along with other issues

**Review by Themperror — CHANGES_REQUESTED** [review]
Please have a look at:
https://github.com/GraphicsProgramming/learnd3d11/blob/main/src/Cpp/1-getting-started/1-HelloWindow/Application.cpp#L53

**Comment by AlexB8675** [comment]
> Please have a look at: https://github.com/GraphicsProgramming/learnd3d11/blob/main/src/Cpp/1-getting-started/1-HelloWindow/Application.cpp#L53

I have, is there something wrong in my commit?

**Comment by Themperror** [comment]
> > Please have a look at: https://github.com/GraphicsProgramming/learnd3d11/blob/main/src/Cpp/1-getting-started/1-HelloWindow/Application.cpp#L53
> 
> I have, is there something wrong in my commit?

Ah sorry I completely overlooked the fact it was the win32 class, I thought you were using WIN32 such as GetScreenMetrics in the glfw code. so nevermind :D

approved

File: src/Cpp/1-getting-started/12-HelloGLFWWindow/Main.cpp [hunk 2]
```diff
@@ -0,0 +1,37 @@
+#include <GLFW/glfw3.h>
+
+#include <iostream>
+
+static void CenterWindow(GLFWwindow* windowHandle)
```
Comment (deccer) [inline src/Cpp/1-getting-started/12-HelloGLFWWindow/Main.cpp:5]: Lets integrate CenterWindow like wave in markdown

File: src/Cpp/1-getting-started/12-HelloGLFWWindow/Main.cpp (hunk 1 above)
Comment (deccer) [reply src/Cpp/1-getting-started/12-HelloGLFWWindow/Main.cpp]: Yeah like the markdown variant it will be taken care of

**Review by deccer — APPROVED** [review]
LGTM

Once we have a decent Markdown structure, we might have to readjust the folder structure a little, but we can do that later, no big deal

**Review by deccer — APPROVED** [review]
<3

**Pull request merged**